tests/foreman/cli/test_activationkey.py

```diff
@@ -30,6 +33,7 @@ class TestActivationKey(CLITestCase):
     product = None
     env1 = None
     env2 = None
+    pub_key = None
```

Comment:
i settle on pub_key (short for public_key) to make it a bit easier to stay within 80 characters.